data/network/src/main/java/io/filmtime/data/network/TmdbMoviesService.kt

```diff
@@ -9,13 +9,11 @@ interface TmdbMoviesService {
 
   @GET("movie/{movie_id}")
   suspend fun getMovieDetails(
-    @Path("movie_id") movieId: Int,
-    @Query("api_key") apiKey: String = BuildConfig.TMDB_API_KEY,
+    @Path("movie_id") movieId: Int
   ): NetworkResponse<TmdbMovieDetailsResponse, TmdbErrorResponse>
 
   @GET("trending/movie/{list_type}")
   suspend fun getMovieList(
-    @Path("list_type") listType: String,
-    @Query("api_key") apiKey: String = BuildConfig.TMDB_API_KEY,
+    @Path("list_type") listType: String
```

Comment:
Please do not remove trailing commas
```suggestion
    @Path("list_type") listType: String,
```